connector/src/main/java/org/geysermc/connector/configuration/GeyserConfiguration.java

```diff
@@ -35,7 +35,7 @@
 public interface GeyserConfiguration {
 
     // Modify this when you update the config
-    int CURRENT_CONFIG_VERSION = 4;
+    int CURRENT_CONFIG_VERSION = 5;
```

Comment:
Historically we have only bumped the config option when there are breaking changes in the config.

Reply:
In that case this comment is bit ambiguous. `// Modify this when you introduce breaking changes into the config` or something

Reply:
Feel free to include that in your PR.